src/main/resources/messages.properties

```diff
@@ -0,0 +1,17 @@
+CREATED=Employee Created
+NO_DATA_FOUND=No Records Found
```

Comment:
Messages can be more informative and more meaningful.
For example if a getEmployee has been called with an employee id which doesn't exit, then user should get message as "Employee with id #1 not found in out record."